src/tool/hpcrun/sample-sources/nvidia.c

```diff
@@ -397,14 +392,7 @@ METHOD_FN(process_event_list, int lush_metrics)
   int device_semaphore_size = 
     control_knob_value_get_int(HPCRUN_CUDA_DEVICE_SEMAPHORE_SIZE);
 
-  if (device_buffer_size == 0) {
-    device_buffer_size = DEFAULT_DEVICE_BUFFER_SIZE;
-  }
-
-  if (device_semaphore_size == 0) {
-    device_semaphore_size = DEFAULT_DEVICE_SEMAPHORE_SIZE;
-  }
-
```

Comment:
These variables are nvidia specific. I would suggest to initialize the variables here.

Reply:
Ok, I turned them back. Should I delete them from knobs?